examples/gnn_link.py

```diff
@@ -50,7 +50,7 @@
 root_dir = "./data"
 
 # TODO: remove process=True once correct data/task is uploaded.
-dataset: RelBenchDataset = get_dataset(name=args.dataset, process=True)
+dataset: RelBenchDataset = get_dataset(name=args.dataset, process=False)
```

Comment:
Does this work for all datasets? For example, I believe HM is not working

Reply:
ah yes. i will upload the other databases too (assuming that the databases are stable even though we are still iterating on the task definitions).

currently database is processed every time the script runs and wastes a lot of time in startup.

Reply:
Right. How about leaving it as `True` until our final release? I usually find it ok (~2min to start). Feel free to make it `False` locally.

Reply:
I have uploaded the other databases. So process=False works correctly. But I will revert to process=True for the public code as you say.

Reply:
Thanks!